tests/std/tests/GH_001105_streambuf/test.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <istream>
+#include <stdexcept>
+
+struct bad_buf : public std::streambuf {

```

Comment:
`struct`s default to `public` for both their base classes and members, and we conventionally rely on this default in both product code and test code:
```suggestion
struct bad_buf : std::streambuf {
```